reports/sklearn_estimators_sample_weight_audit_report.ipynb

```diff
@@ -947,11 +1176,11 @@
     "    \"failed the deterministic test\"\n",
```

Comment:
This code should also be simplified since it's no longer reachable if we no longer run the deterministic test.